examples/colour-uniform/README.md

```diff
@@ -0,0 +1,6 @@
+# Colour Uniform
+
+The following image is an output of `cargo run --bin colour-uniform --features=gl` with R set at 127,
+B set at 100, and G set at 50:
+
+![colour_uniform](colour_uniform_screenshot.png "Colour Uniform")
```

Comment:
does the name need to include the example name? since it's already in the folder, it seems that we could just call it "screenshot.png" or something.

Reply:
I guess it's a matter of taste. I prefer to be explicit in naming, but I'm more than happy to change the filename to screenshot.png if necessary. :smiley:

Reply:
I mean, we aren't calling the other files "colour_uniform_cargo.toml" etc :)
If you don't mind, let's rename these to just "screenshot.png" then.

Reply:
Good point! I've updated the filenames.

Reply:
Anything else needed from me?